setup.py

```diff
@@ -21,7 +21,7 @@
 hyperopt = [
     'scipy',
     'scikit-learn',
-    'scikit-optimize',
+    'scikit-optimize>=0.7.4',
```

Comment:
Unless it's been fixed in 0.7.4, i would use:

```suggestion
    'scikit-optimize>=0.7.0',
```

Requirements in setup.py shold be as loose as possible ... requirements.txt (and consorts) on the other hand  pin the latest version (mainly for CI ).

Reply:
pls review the condition: the version of skopt where it was fixed is numbered as 0.7, will comparison `>=0.7.0` work with `0.7`? (assume yes, but anyway...)